config/source/configmaps/leader-election.yaml

```diff
@@ -15,8 +15,8 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: config-leader-election-kafka
-  namespace: knative-sources
+  name: config-leader-election-kafka-source
```

Comment:
This will leave duplicates upon the next deployment, do we want to introduce that here?

Reply:
Good question, @antoineco

Reply:
I think we should avoid renaming things in this PR if they are unrelated to the original intention which is "Move the Kafka Source to the knative-eventing namespace".